src/Service/Resolvers/SqlMutationEngine.cs

```diff
@@ -347,7 +347,7 @@ public class SqlMutationEngine : IMutationEngine
                         throw new DataApiBuilderException(
                             message: "Could not insert row with given values.",
                             statusCode: HttpStatusCode.Forbidden,
-                            subStatusCode: DataApiBuilderException.SubStatusCodes.AuthorizationCheckFailed
+                            subStatusCode: DataApiBuilderException.SubStatusCodes.DatabasePolicyFailure
```

Comment:
Here is an example of status code 403 due to databasepolicyfailure

Reply:
Are you just quoting an example or suggesting some change? I'm thinking the former.